SQLite/Value.swift

```diff
@@ -62,6 +62,10 @@ public struct Blob {
     public init(bytes: UnsafePointer<Void>, length: Int) {
         data = NSData(bytes: bytes, length: length)
     }
+    
+    public init (data: NSData) {
```

Comment:
Any chance we can get rid of this convenience initializer? The `swift-2` branch actually uses a fully Swift implementation of `Blob` (no reliance on `Foundation`/`NSData`), which I may back-port, at which point you can still create a blob from `NSData` like this:

``` swift
// data: NSData
let blob = Blob(bytes: data.bytes, length: data.length)
```